packages/blockchain-wallet-v4-frontend/src/services/CoinifyService/index.js

```diff
@@ -246,3 +247,22 @@ export const getReasonExplanation = (reason, time) => {
     default: return <FormattedMessage id='scenes.coinify.cannottradereason.unknown' defaultMessage='Trading is disabled.' />
   }
 }
+
+export const recurringTimeHelper = (sub) => {
+  let human = { 1: 'st', 2: 'nd', 3: 'rd', 21: 'st', 22: 'nd', 23: 'rd', 31: 'st' }
+  let days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']
+
+  let getTimespan = (sub) => {
+    let frequency = prop('frequency', sub)
+    let freq = frequency.toLowerCase()
+    let date = new Date()
+
+    if (freq === 'hourly') return 'hour'
+    if (freq === 'daily') return '24 hours'
+    if (freq === 'weekly') return `${days[date.getDay()]}`
+    if (freq === 'monthly') return `${date.getDate() + (human[date.getDate()] || 'th')} of the month`
```

Comment:
Consider using a switch statement here

Reply:
Good point, this was copy/pasted over from V3.